server/message_hooks.go

```diff
@@ -40,111 +40,110 @@ func (p *Plugin) MessageHasBeenDeleted(_ *plugin.Context, post *model.Post) {
 	if p.getConfiguration().UseSharedChannels {
 		return
 	}
-	p.messageDeletedHandler(post)
+	_ = p.messageDeletedHandler(post)
```

Comment:
For me was ideal to introduce it here because I did want to give also that support for shared channels, we can reintroduce it later if the PR is reverted, not a big deal, IMO.